src/app/validation/config/config.exs

```diff
@@ -1,5 +1,10 @@
 import Config
 
+# BIG ISSUE with this method: private key gets fed in as start_link args to goth, so it shows up in sasl logs
+# TODO: custom credentials module
+config :goth,
+  json: System.get_env("GOOGLE_CLOUD_AUTOMATED_VALIDATION_KEYFILE_JSON") |> File.read!()
```

Comment:
This should go in your local `local.exs` only (so that other's can configure their own way of loading this).